### ISSUE TITLE
Placeholder / Nested Mathfield Incorrect Position on First Render

### ISSUE THREAD

**Issue by gerryfletch** [arnog/mathlive#1771]
## Description
- In recent updates, the nested mathlive `placeholder` fields have a graphical bug where on first render the position is incorrect.
- Clicking the mathfield/placeholders seems to cause a rerender, which fixes the bug.
- I expect the CSS properties are calculated programmatically and clicking on the mathfield retriggers it.
- Occasionally, the first render is correct, but frequently it is not

## Steps to Reproduce
- This bug is actually live on the docs, see: https://cortexjs.io/mathlive/guides/interacting/#fill-in-the-blank
- Before clicking:

<img width="244" alt="image" src="https://user-images.githubusercontent.com/24813360/210173146-362f8341-00b8-40d7-bf76-9b353c9b5e95.png">

- After clicking:
<img width="243" alt="image" src="https://user-images.githubusercontent.com/24813360/210173157-caac2fcf-d63b-4db6-8bb5-24cadfe20d1c.png">

- Here is my personal use of placeholder with the bug:

<img width="763" alt="image" src="https://user-images.githubusercontent.com/24813360/210173216-b0b0757d-7f94-4aab-9bc8-12f07a97b5db.png">

### Expected Behavior
- The placeholder should be in the correct place on first render

### Environment
- Version: latest (0.86.0)
- OS: Mac OS
- Browser: Chrome

**Comment by gerryfletch** [comment]
On the first render, the nested `math-live` element has the following style attribute `top: 31px`:

```html
<math-field virtual-keyboard-mode="off" role="textbox" dir="ltr" aria-label="math input field" contenteditable="true" aria-multiline="false" tabindex="0" class="placeholder-mathfield" style="left: 89.4375px; top: 31px;">
```

Focusing the box updates the style to be `top: 12px`, which is correct:

```html
<math-field virtual-keyboard-mode="off" role="textbox" dir="ltr" aria-label="math input field" contenteditable="true" aria-multiline="false" tabindex="0" class="placeholder-mathfield" style="left: 89.4375px;top: 12px;">
```

**Comment by volesen** [comment]
Do we expect, in this case, that the x's are baseline aligned?

**Comment by WildYellowfin** [comment]
@arnog any update on this?

**Comment by arnog** [comment]
@WildYellowfin it's on my todo list, but I probably won't be able to get to it for another two weeks.

**Comment by arnog** [comment]
Fixed in mathlive 0.90.0

**Comment by gerryfletch** [comment]
Fixed! Nice one, thanks all
